### PR TITLE
Milestone 2: Models

### PR DESCRIPTION
**Pull request by AbigiyaTY**
**Building Portfolio page milestone 2: Models**

In this milestone I:

- Build the work display section
- Build the contact us section
- Added styling to both section

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -160,6 +158,200 @@ a {
...
+  background-image: url('../images/Ellipse-5.svg');
+  background-position: left;
+}
+
+.work-portofolio-4 {
+  background-image: url('../images/Ellipse-6.svg');
+  background-position: right;
+}
+
 @media (min-width: 375px) {
```
Comment (isaacpitwa) [inline css/style.css]: Great work implementing the desktop version 👏🏽
However, your design is not visible on most mobile devices i.e devices width > 375px.

- [ ] Kindly consider expanding the range of mobile devices to cater to mobile devices  as the  maximum mobile screen size is  480px . Check the screenshots below.
Screenshot 2022-07-02 at 00 11 54
Screenshot 2022-07-02 at 00 11 43

File: index.html [hunk 2]
```diff
@@ -24,83 +26,209 @@ <h3 class="name1">AB</h3>
...
-                <h3>Language</h3>
-                <ul>
-                    <li>Javascript</li>
-                    <li>Ruby</li>
-                    <li>Html</li>
-                    <li>Css</li>
-                </ul>
+    <section>
+        <div class="work">
+            <h1 class="work-title">MY RECENT WORK</h1>
```
Comment (isaacpitwa) [inline index.html]: - [ ] Kindly consider having a single h1 in a webpage. h1 is used to indicate the most important heading (for this case, your name), having more than one may lead to spamming. kindly change this to one of the least important headings.

**Review by isaacpitwa — CHANGES_REQUESTED** [review]
Hi @AbigiyaTY  🙂 ,

You have done a tremendous job setting up your project. I really like it :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

Alt Text


## Highlights 
- All linters are passing  ✅
- Correct use of GitHub flow ✅
- Well documented Repository ✅
-  Clean and readable code base ✅

## Required changes ♻️ 

Check the comments under the review.

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Check the comments under the review.





Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @isaacpitwa.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by danielufeli — APPROVED** [review]
Hi @AbigiyaTY ,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

a

## Highlights
- Nice commits messages✔️
- Descriptive PR ✔️
- Good use of GitHub flow ✔️
- Clean Code ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @danielufeli in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**